ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryTopComponent.java

```diff
@@ -400,6 +443,15 @@ public void componentOpened() {
 
     @Override
     public ExplorerManager getExplorerManager() {
+        /*
+         * Although ImageGallery doesn't currently use the explorer manager,
+         * this TopComponent provides one through the getExplorerManager method.
+         * However, this does not seem to function correctly unless a Netbeans
+         * provided explorer view is present in the TopComponenet, even if it is
+         * invisible/ zero sized
+         */
+        // RJCTODO: Why is this override here? Does the "this" in "this does 
+        // not seem to function correctly" refer to the methdo or the top compnent?
```

Comment:
it refers to the method/explorer manager.

Reply:
this comment seems partly obsolete.  The explorer manager is used by the FileIDSelectionModel to enable reuse of autopsy actions in ImageGallery.

I believe the part about not working unless there is a Netbeans supplied explorer view is present is still correct.

Reply:
Thanks for the clarification. I will remove the obsolete comment and my TODO.